README.md

```diff
@@ -1,88 +1,114 @@
-# Svelte Demo
+# Web Component Svelte Demo
 
 ![Travis (.org)](https://img.shields.io/travis/alaskaairlines/AuroSvelteDemo?style=for-the-badge) ![GitHub tag (latest by date)](https://img.shields.io/github/v/tag/AlaskaAirlines/AuroSvelteDemo?style=for-the-badge)
 
-Interested in starting a new project using Auro and Svelte? This demo project is a complete example that includes basic setup. 
+An example Svelte project with Auro Components integrated. This app runs and works in all Alaska-supported browsers. Explore the project source to see the Auro Components being used in a frameworkless environment.
 
-To locally run this demo: 
-
-```bash
-$ git clone https://github.com/AlaskaAirlines/AuroSvelteDemo.git
-```
-
-The app runs and works in all [supported browsers](http://auro.alaskaair.com/support/browsersSupport). Please explore the project source to see how the Auro components are being used in a Svelte development environment. 
-
-This is a project template for [Svelte](https://svelte.dev) apps. It lives at [https://github.com/sveltejs/template](https://github.com/sveltejs/template).
-
-To create a new project based on this template using [degit](https://github.com/Rich-Harris/degit):
+Below are instructions for adding Auro compatibility to your project. A familiarity with modern Javascript development and bundlers such as Webpack is expected.
 
 ## Example App API
 
-This project is configured to work with `npm`. Within the root directory of the app, you can:
+Depending on your local configuration, this project will work with either `yarn` or `npm`. For the sake of documentation, the commands will default to `npm`.
+
+Within the root directory of the app, you can:
 
 | Command | Description
 |---|---
-| npm start | Runs the app in the development mode.<br />Open [http://localhost:5000](http://localhost:5000) to view it in the browser.
-| npm run build | Production build in the `build` folder.
-| npm run dev | Runs dev server with browser sync enabled.
+| npm start | Runs the app in the development mode.<br />Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+||Browser sync and linting in the CLI is enabled.
+| npm test | Launches the test runner in the interactive watch mode
+| npm build | Production build in the `build` folder.<br>Optimized, minimized and prepared for deployment
```

Comment:
We should remove `npm test` since we don't have any tests and also mention `npm run dev`.